src/components/common/Extensions/UserHub/partials/ReputationTab/partials/PendingReputation/PendingReputation.tsx

```diff
@@ -17,6 +17,8 @@ import {
 const displayName =
   'common.Extensions.UserHub.partials.ReputationTab.partials.PendingReputation';
 
+const UPDATE_INTERVAL = 15;
```

Comment:
I'd put this same as the block time (from constants default network) but I guess any value will work in the end, as the update estimate is just that, and estimate.